README.reproducible.md

```diff
@@ -0,0 +1,167 @@
+Linux.
+
+    cd path-to-armory-source
+    ./autogen.sh
+    ./configure # don't need to specify host if building on same architecture as host
+    make
+    make install # optional
+
+Specify `--host x86_64-linux-gnu` or `--host i686-linux-gnu` if you are
+building for 64-bit from a 32-bit machine or building 32-bit from a 64-bit
+machine respectively.
+
+If you don't run `make install`, you can just run `python ArmoryQt.py` from the
+Armory source tree. If you run `make install`, you should have `armory` on your
+path. So you can just run `armory`.
+
+###Fedora Package (RPM)
+
+RPM packages are not officially supported. They are a convenience to users.
+Read [rpmfiles/armory.spec.README](rpmfiles/armory.spec.README) for more info.
+
+##OS X
+
+###Reproducible
+
+The OS X reproducible build system uses Gitian. Refer to the Gitian
+instructions and use gitian-descriptors/bitcoin-armory-osx.yml as the Gitian
+descriptor.
+
+###Non-reproducible
+
+You can use the Autotools-based build system to build Armory.app. This has been
+tested on OS X and Linux as the build machines.
+
+For OS X:
+
+    cd path-to-armory-source
+    ./autogen.sh
+    ./configure
+    make Armory.app
+
+For Linux:
+
+    # install cctools-port, TODO: add instructions for installing cctools-port
+    echo -e '#!/bin/sh\nexit 0' > /usr/local/bin/x86_64-apple-darwin11-dsymutil
+    mv /usr/local/bin/x86_64-apple-darwin11-ld /usr/bin/
+    cd path-to-armory-source
+    ./autogen.sh
+    ./configure --host=x86_64-apple-darwin11 \
+                CC="clang -target x86_64-apple-darwin11 -mmacosx-version-min=10.7 --sysroot /usr/share/MacOSX10.9.sdk -mlinker-version=241.9" \
+                CXX="clang++ -target x86_64-apple-darwin11 -mmacosx-version-min=10.7 --sysroot /usr/share/MacOSX10.9.sdk -mlinker-version=241.9"
+    make Armory.app
+
+In either case, you should be left with an Armory.app in osxbuild that will run
+on OS X 10.7+.
+
+Alternatively, you can just run `make` if you don't want to build Armory.app.
+
+Note that Armory.app currently builds with Python 3 and Qt 5, but Armory still
+uses Python 2 and Qt 4. Therefore, the user needs to install Qt 4 before
+Armory.app will run.
+
+##Windows
+
+###Reproducible
+
+The new reproducible build system does not currently support Windows builds.
+Work is being done to make a single static exe that includes everything needed
+to run Armory on a Windows machine without requiring the end use to install
+Python or anything else. Just double-click on the exe and Armory pops up.
+
+Right now we need to be able to build Twisted linked against the Python that
+we build. But there is an [issue with Wine][Wine Issue] that is preventing us
+from being able to build Twisted from a Linux machine. We want to be able to
+do the entire build from the same Linux machine, so the Windows build is halted
+until the Wine issue is resolved.
+
+###Non-reproducible
+
+If you just want to build a non-reproducible version of Armory for Windows, you
```

Comment:
See the OS X comment. I think the directions under Windows_Build_Instructions.rtf are still valid. If not, I'll patch this up later.